server/email_verification.go

```diff
@@ -215,6 +217,96 @@ func handleEmailVerifyFunc(verifiedTpl *template.Template, issuer url.URL, keysF
 			return
 		}
 
+		aud, ok, err := ev.Claims.StringClaim("aud")
+		if !ok {
+			execTemplateWithStatus(w, verifiedTpl, emailVerifiedTemplateData{
+				Error:   "Unable to get 'aud' from token",
```

Comment:
Since there's a huge amount of duplicated error handling here, you'll definitely want to break this into a separate function to get rid of the

```
execTemplateWithStatus(w, verifiedTpl, emailVerifiedTemplateData{

}, code)
```
Repetition.